GoogleForCreators/web-stories-wp
# Safari content stacking fix for border radius

## Summary
Changes the content stacking for fixing border radius for Safari.


## Relevant Technical Choices



## To-do



## User-facing changes
|Before|After|
|-----|-----|
| ![before](https://user-images.githubusercontent.com/3294597/121878079-96024e80-cd0b-11eb-8eda-02bfb5017711.gif) | ![after-](https://user-images.githubusercontent.com/3294597/121878091-9995d580-cd0b-11eb-952f-fffdd409ea5b.gif) |


## Testing Instructions
In Safari / iPhone 11 
1. Add a square shape (or media)
2. Set border radius and border; add animation
3. Save the story
4. View it in the front-end and verify the border radius is displayed as expected and animation works, too.


### QA
See above
## Checklist




---



Fixes #7524
